app/services/manifest_builder.rb

```diff
@@ -605,7 +620,7 @@ def manifest_image_thumbnail_path(resource)
     end
 
     def manifest_image_medium_path(resource)
-      "#{manifest_image_path(resource)}/full/!1000,/0/default.jpg"
+      "#{manifest_image_path(resource)}/full/1000,/0/default.jpg"
```

Comment:
This was necessary for me when I was testing using `riiif`, and I could not find a solution for resolving this within my local development environment.